ReleaseNotes: Updated changelog and release notes for 8.2.0-beta1 (#39291)
grafana/grafana: docs/sources/release-notes/release-notes-8-2-0-beta1.md

Panel queries and/or annotation queries that used more than one statistic will be converted into one query/annotation per statistic. In case an alerting rule was based on a query row that had more than one statistic, it would now be based only on the first statistic for that query row. New alerting rules will not be created for migrated queries. Please note that in most cases it would not make sense to have an alerting rule that is based on multiple statistics anyway.  Issue [#36925](https://github.com/grafana/grafana/issues/36925)

### Deprecations

`getHighlighterExpressions` in datasource APIs ( used to highlight logs while editing queries) has been deprecated and will be removed in a future release.

# Deprecation notice

`ExploreQueryFieldProps` interface for query editors has been deprecated and will be removed in a future release. Use `QueryEditorProps` instead. Issue [#38942](https://github.com/grafana/grafana/issues/38942)

### Plugin development fixes & changes

- **Grafana UI:** Fix TS error property `css` is missing in type. [#38078](https://github.com/grafana/grafana/pull/38078), [@jackw](https://github.com/jackw)

> 

Reviewer: ```suggestion

### Grafana 8 alerts fixes

- **Organisation level isolation** The fix for organisation level isolation [introduces a new migration](https://github.com/grafana/grafana/pull/37414) that deletes all Grafana 8 alerting data and regenerates them so that on Alertmanager configuration is created properly for each organisation. This has as side-effect that any rules or Alertmanager configuration created or modified manually after enabling the feature will be removed and recreated.
```

Reviewer 2: :+1: thanks

- **Grafana UI:** Fix TS error property `css` is missing in type. [#38078](https://github.com/grafana/grafana/pull/38078), [@jackw](https://github.com/jackw)

### Grafana 8 alerts fixes

> - **Organisation level isolation:** The fix for organisation level isolation [introduces a new migration](https://github.com/grafana/grafana/pull/37414) that deletes all Grafana 8 alerting data and migrates the dashboard alerts to the new system again so that an Alertmanager configuration is created properly for each organisation. As a result, any manually created (or modified) Grafana 8 alerts or Alertmanager configuration will be removed after upgrading.

Reviewer 3: ```suggestion
- **Organisation level isolation:** The fix for organization level isolation introduced a [new migration flow](https://github.com/grafana/grafana/pull/37414) that deletes all Grafana 8 alerting data and migrates the dashboard alerts to the new system again so that an Alertmanager configuration is created properly for each organization. It removes any manually created (or modified) Grafana 8 alerts or Alertmanager configuration after upgrading.
```
